definitions/tools/kallisto.cwl

```diff
@@ -3,34 +3,46 @@
 cwlVersion: v1.0
 class: CommandLineTool
 label: "Kallisto: Quant"
-baseCommand: ["/usr/bin/kallisto"]
+baseCommand: ["/bin/bash","kallisto.sh"]
 requirements:
     - class: ResourceRequirement
       ramMin: 32000
       coresMin: 8
     - class: DockerRequirement
       dockerPull: "mgibio/rnaseq"
+    - class: InitialWorkDirRequirement
+      listing:
+      - entryname: 'kallisto.sh'
+        entry: |
+            set -eou pipefail
+
+            ncores="$1"
+            index="$2"
+            strand="$3"
+            paired="$4"
+
+            #fragment params are ignored and inferred from the data for paired-end
+            fragment_length="$5"
+            fragment_length_stddev="$6"
+
+            fastqs="${@:7}"
+
+            if [[ "$paired" == "true" ]];then
+              /usr/bin/kallisto quant -t $ncores -b 100 --fusion -o kallisto -i $index $strand $fastqs
+            else
+              ##"Reasonable" values for fragment length and stddev are hardcoded here. If your input data is substantially different
```

Comment:
This comment is confusing to me, since the below `kallisto quant` call has the value of `-b` hardcoded but not `-l` or `-s`.